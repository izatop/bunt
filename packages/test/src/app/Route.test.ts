import HelloWorldRoute from "./src/actions/HelloWorldRoute";

test("Route", async () => {
    expect(HelloWorldRoute).toMatchSnapshot();
});
