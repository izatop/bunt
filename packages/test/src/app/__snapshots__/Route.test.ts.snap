// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route 1`] = `
PathRoute {
  "action": [Function],
  "route": "/u/:id",
  "state": Object {
    "id": [Function],
    "payload": [Function],
  },
  "validate": undefined,
}
`;
